docs/configuration.md

```diff
@@ -274,6 +274,10 @@ storage:
 The storage option is **required** and defines which storage backend is in use.
 You must configure one backend; if you configure more, the registry returns an error.
 
+If you are deploying a registry on Windows, Docker strongly advises that you do not use a Windows volume mounted from the host. Instead, you can use a S3, or Azure, backing data-store. If you do use a Windows volume, you must ensure that the `PATH` to the mount point is within Window's `MAX_PATH` limits. Failure to do so can result in the following error message: 
```

Comment:
"Docker" is not advising anyone. This should be passive "It is advised" or "Providence advises":

```
If deploying the registry on Windows, it is strongly advised to *not* use a Windows volume mounted from the host.
```